noether_examples/launch/mesh_segmenter_node.launch

```diff
@@ -0,0 +1,19 @@
+<launch>
+  <arg name="filename"/>
+  <arg name="min_cluster_size" default="500"/>
+  <arg name="max_cluster_size" default="1000000"/>
+  <arg name="curvature_threshold" default="0.3"/>
+  <arg name="show_individually" default="false"/>
+  <arg name="save_outputs" default="false"/>
+
+  <node name="mesh_segmenter_node" type="mesh_segmenter_node" pkg="noether_examples" output="screen" required="true" >
+    <!--Loads a particular test file: pcd or stl -->
```

Comment:
see, these are private parameters. the ones outside  the node are global parameters. Your code accesses private parameters as if they were global. Its not usual.